website/package.json

```diff
@@ -15,13 +15,14 @@
     "@babel/core": "^7.7.2",
     "@docusaurus/core": "2.0.0-alpha.35",
     "@docusaurus/preset-classic": "2.0.0-alpha.37",
-    "@icgc-argo/uikit": "^1.0.9",
+    "@icgc-argo/uikit": "^1.0.1-0.alpha.ciaran.7",
```

Comment:
Should I ask what this version naming scheme is?

Reply:
@joneubank ah yes! local npm package of uikit isn't working. So to test new uikit changes working in argo docs, just using made up naming for my testing.

Once it's approved in argo-platform repo, uikit should be published using our standard naming

Does this sound ok to you?

Reply:
sounds good. i merged the PR in argo-platform just now... hopefully that didn't ruin the flow you had planned 😟

Reply:
all good i think!